frontend/simple/components/Proposals/Voting.vue

```diff
@@ -0,0 +1,290 @@
+<template>
+<div class="is-flex gi-voting">
```

Comment:
This should be indented for sure though (since we're dealing with XML inside XML)